### PR TITLE
Fix local function issue for variables.env

### LINKED ISSUE #40: Error on sourcing .bingo/variables.env files
`.bingo/variables.env` is using `local` outside a function it causes an error on certain systems.

```console
$ source .bingo/variables.env
.bingo/variables.env: line 4: local: can only be used in a function
```

See also https://app.circleci.com/pipelines/github/thanos-io/thanos/2025/workflows/74888d11-3a8e-413b-83e5-0a4928b4d59e/jobs/10145

### PR DESCRIPTION
**Pull request by kakkoyun**
Signed-off-by: Kemal Akkoyun <kakkoyun@gmail.com>

Fixes #40 

cc @periklis @bwplotka

### CODE REVIEW

**Review by periklis — APPROVED** [review]
LGTM

File: .bingo/variables.env [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+FAILLINT="$GOBIN/faillint-v1.5.0"
 
-GOIMPORTS="${gobin}/goimports-v0.0.0-20200519204825-e64124511800"
+GOIMPORTS="$GOBIN/goimports-v0.0.0-20200519204825-e64124511800"
 
-GOLANGCI_LINT="${gobin}/golangci-lint-v1.26.0"
+GOLANGCI_LINT="$GOBIN/golangci-lint-v1.26.0"
 
-MISSPELL="${gobin}/misspell-v0.3.4"
+MISSPELL="$GOBIN/misspell-v0.3.4"
```
Comment (bwplotka) [inline .bingo/variables.env:21]: Can we use quote safe ${} for all vars?

**Review by bwplotka — APPROVED** [review]
Thanks! One suggestion only

**Review by bwplotka — APPROVED** [review]
Sorry fo lag :scream:

**Pull request merged**